draft-yasskin-wpack-bundled-exchanges.md

```diff
@@ -383,7 +383,10 @@ map, and the `metadata` map to fill in, the parser MUST do the following:
 ### Parsing the manifest section {#manifest-section}
 
 The "manifest" section records a single URL identifying the manifest of the
-bundle. The bundle can contain multiple resources at this URL, and the client is
+bundle. The URL must refer to the one or more response(s) contained in the
```

Comment:
"MUST" should be capitalized per https://tools.ietf.org/html/rfc8174.

Reply:
Done.